python/MANIFEST.in

```diff
@@ -17,6 +17,8 @@
 global-exclude *.py[cod] __pycache__ .DS_Store
 recursive-include deps/jars *.jar
 graft deps/bin
+graft deps/data
```

Comment:
On a second thought I think it would be better to keep all the packages consistent (specifying the patterns to include in both MANIFEST.in and setup.py), so I changed the `graft` to `recursive-include` with proper patters as you suggested.